src/cli.ts

```diff
@@ -76,69 +76,118 @@ Provide it via the ${AUTOBLOCKS_API_KEY_NAME} environment variable, within a .en
-          type: 'boolean',
-          default: false,
-        })
-        .option('slack-webhook-url', {
-          describe: `Slack webhook URL where results are posted. Can be set via the ${AUTOBLOCKS_SLACK_WEBHOOK_URL_NAME} environment variable.`,
-          type: 'string',
-          default: variableFromEnv(AUTOBLOCKS_SLACK_WEBHOOK_URL_NAME),
-        })
-        .demandOption('api-key', apiKeyMissingErrorMessage)
-        .help();
-    },
-    (argv) => {
-      const errorMessage = `No command found. Provide a command following '--'. For example:
-
-npx autoblocks testing exec -- echo "Hello, world!
+const makeTestingCommandNotFoundErrorMessage = (command: string) => `
+No command found. Provide a command following '--'. For example:
+
+npx autoblocks testing ${command} -- echo "Hello, world!
 `;
 
-      const unparsed = argv['--'];
-      if (!Array.isArray(unparsed)) {
-        throw new Error(errorMessage);
-      }
-
-      const [command, ...commandArgs] = unparsed.map(String);
-
-      if (!command) {
-        throw new Error(errorMessage);
-      }
-
-      handlers.testing.exec({
-        command,
-        commandArgs,
-        apiKey: argv['api-key'],
-        runMessage: argv.message,
-        port: argv.port,
-        exit1OnEvaluationFailure: argv['exit-1-on-evaluation-failure'],
-        slackWebhookUrl: argv['slack-webhook-url'],
-      });
-    },
-  )
+const parseCommandFromArgv = (args: {
+  command: string;
+  argv: yargs.Arguments;
+}): { command: string; commandArgs: string[] } => {
+  const unparsed = args.argv['--'];
+  if (!Array.isArray(unparsed)) {
+    throw new Error(makeTestingCommandNotFoundErrorMessage(args.command));
+  }
+
+  const [command, ...commandArgs] = unparsed.map(String);
+
+  if (!command) {
+    throw new Error(makeTestingCommandNotFoundErrorMessage(args.command));
+  }
+
+  return { command, commandArgs };
+};
+
+const apiKeyOptions = {
+  describe: `Autoblocks API key. Can be set via the ${AUTOBLOCKS_API_KEY_NAME} environment variable.`,
+  type: 'string',
+  default: variableFromEnv(AUTOBLOCKS_API_KEY_NAME),
+  demandOption: apiKeyMissingErrorMessage,
+} as const;
+
+const portOptions = {
+  alias: 'p',
+  describe: 'Local CLI server port number',
+  type: 'number',
+  default: 5555,
+} as const;
+
+const parser = yargs(hideBin(process.argv))
+  .command('testing', 'Autoblocks Testing', (yargs) => {
+    return yargs
+      .command(
+        'exec',
+        'Execute a command that runs your Autoblocks test suites',
+        (yargs) => {
+          return yargs
+            .option('api-key', apiKeyOptions)
+            .option('message', {
+              alias: 'm',
+              describe: 'Description for this test run',
+              type: 'string',
+            })
+            .option('port', portOptions)
+            .option('exit-1-on-evaluation-failure', {
+              describe: 'Exit with code 1 if any evaluation fails.',
```

Comment:
nit: Consistency with punctuation (using . at end of second describe in highlighted block)